spec/system/post_selection_copy_quote_spec.rb

```diff
@@ -30,7 +30,7 @@
       topic_page.visit_topic(topic)
 
       select_text_range("#{topic_page.post_by_number_selector(1)} .cooked p", 0, 10)
-      expect(page).not_to have_css(topic_page.copy_quote_button_selector)
+      expect(page).to have_no_css(topic_page.copy_quote_button_selector)
```

Comment:
not related to the fix, but this is a better selector and is in line with what I did for the new test